src/xcode/ENA/ENA/Source/Scenes/Home/HomeInteractor.swift

```diff
@@ -431,8 +431,15 @@ extension HomeInteractor {
 
 extension HomeInteractor {
 	func updateTestResults() {
+		
+		// Do warn others evaluation
+		if let testResult = testResult {
+			self.warnOthers.evaluateNotificationState(testResult: testResult)
```

Comment:
Should the notification also be triggered from the home view?

Reply:
Yes. I explicitly double checked this with Max.